### PR TITLE
Add i18n check action

### PR DESCRIPTION
**Pull request by shun2wang**

### CODE REVIEW

File: .github/workflows/i18nCheck.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+on: 
+  push:
+    paths: ['**.R', '**.qml', 'tests/**', 'inst/**', '**.Rd', '**.c', '**.cpp', '**.h', '**.hpp', '**.yml']
+ 
+  pull_request:
+    paths: ['**.R', '**.qml', 'tests/**', 'inst/**', '**.Rd', '**.c', '**.cpp', '**.h', '**.hpp', '**.yml']
```
Comment (vandenman) [inline .github/workflows/i18nCheck.yml:6]:
```suggestion
    paths: ['**.R', '**.qml', '**.yml']
 
  pull_request:
    paths: ['**.R', '**.qml', '**.yml']
```
is there any reason to trigger this for other file types? As far as I know, only R and qml are translated.

Comment (shun2wang) [reply .github/workflows/i18nCheck.yml:6]: yes, you are right.

**Review by vandenman — APPROVED** [review]

**Comment by shun2wang** [comment]
I am adding this action for all analysis modules:-)
but this PR just for this repo.

**Comment by vandenman** [comment]
> It would be better if the core team member could push directly to all analysis repositories.

perhaps @RensDofferhoff has time to do this? :sweat_smile:

**Review by boutinb — APPROVED** [review]

**Comment by boutinb** [comment]
Yes, we will add this file directly in the modules. No need to make a PR.

**Pull request merged**